database/testing/dqlitesuite.go

```diff
@@ -63,11 +65,26 @@ func (s *DqliteSuite) SetUpTest(c *gc.C) {
 	url := fmt.Sprintf("%s:%d", "127.0.0.1", port)
 	c.Logf("Opening dqlite db with: %v", url)
 
+	// Depending on the verbosity of the test suite, we want to
+	// also print all the sql hitting the db.
+	var verbose bool
+	flag.VisitAll(func(f *flag.Flag) {
+		if verbose || !strings.Contains(f.Name, "check.vv") {
+			return
+		}
+		verbose, _ = strconv.ParseBool(f.Value.String())
+	})
+
 	s.dqlite, err = app.New(s.dbPath,
 		app.WithAddress(url),
 		app.WithTracing(client.LogDebug),
 		app.WithLogFunc(func(level client.LogLevel, msg string, args ...any) {
 			switch level {
+			case client.LogDebug:
+				if !verbose {
+					return
+				}
+				fallthrough
```

Comment:
Do we want to keep this, it was actually useful to understand what was actually happening.